index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Evans_Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="navigation">
+      <div class="logo">
+        <h3>Evans</h3>
+      </div>
```

Comment:
- [ ] Kindly ensure that your logo is wrapped in an `<a>` tag, this is good for convention, semantics and user experience (UX) reasons. 👍